ganttproject/src/biz/ganttproject/impex/google/GoogleExportOptionPageProvider.java

```diff
@@ -0,0 +1,55 @@
+package biz.ganttproject.impex.google;
+
+import biz.ganttproject.core.option.GPOptionGroup;
+import net.sourceforge.ganttproject.gui.options.OptionPageProviderBase;
+import net.sourceforge.ganttproject.gui.options.OptionsPageBuilder;
+import net.sourceforge.ganttproject.language.GanttLanguage;
+
+import javax.swing.*;
+import javax.swing.border.EmptyBorder;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.io.IOException;
+
+public class GoogleExportOptionPageProvider extends OptionPageProviderBase {
+
+  public GoogleAuth googleAuth = new GoogleAuth();
+
+  public GoogleExportOptionPageProvider() {
+    super("impex.google");
+  }
+
+  @Override
+  public GPOptionGroup[] getOptionGroups() {
+    return new GPOptionGroup[0];
+  }
+
+  @Override
+  public boolean hasCustomComponent() {
+    return true;
+  }
+
+  public Component buildPageComponent() {
+    final Action authAction = new AbstractAction() {
+      {
+        putValue(Action.NAME, GanttLanguage.getInstance().getText("googleAuth"));
```

Comment:
if you don't plan to keep this action in the real code, no need to use i18n. If you do plan to have a button in the production UI, use `GPAction` class. Its constructor takes i18n key and fetches the required messages automatically.